Add Boss Health Indicators plugin

rugg0064 opened the pull request:
Boss Health Indicators is a basic plugin that adds markers on boss health bars.

![ingame](https://user-images.githubusercontent.com/35048262/233190712-c464fe8b-a6a6-41a3-8899-fc4ee5536ee9.png)
![panel](https://user-images.githubusercontent.com/35048262/233190718-35a7f315-dc31-4798-be53-4bfebeebf29a.png)

Nightfirecat commented:
Where are your icons sourced from? Are they licensed such that we can distribute them?

Nightfirecat commented:
Also, your plugin should remove the bars when it is shut down so they don't persist when it's disabled.

rugg0064 commented:
> Also, your plugin should remove the bars when it is shut down so they don't persist when it's disabled.

The image is from the wiki. However, I couldn't recall the exact image used. I went ahead and remade it with a wiki image and included a link on the readme. I asked the wiki folks, they said the image is fine to use.

> Also, your plugin should remove the bars when it is shut down so they don't persist when it's disabled.

I have removed the widgets on plugin shut down.


Additionally, changed the priority of the sidebar icon.
